keychain.go

```diff
@@ -144,23 +140,7 @@ func (k *keychain) Set(item Item) error {
 		kcItem.SetAccessible(gokeychain.AccessibleWhenUnlocked)
 	}
 
-	isTrusted := k.isTrusted && !item.KeychainNotTrustApplication
-
-	if isTrusted {
-		debugf("Keychain item trusts keyring")
-		kcItem.SetAccess(&gokeychain.Access{
-			Label:               item.Label,
-			TrustedApplications: nil,
-		})
-	} else {
-		debugf("Keychain item doesn't trust keyring")
-		kcItem.SetAccess(&gokeychain.Access{
-			Label:               item.Label,
-			TrustedApplications: []string{},
-		})
-	}
-
-	debugf("Adding service=%q, label=%q, account=%q, trusted=%v to osx keychain %q", k.service, item.Label, item.Key, isTrusted, k.path)
+	debugf("Adding service=%q, label=%q, account=%q, trusted=%v to osx keychain %q", k.service, item.Label, item.Key, k.path)
```

Comment:
It appears `trusted=%v` should be removed from the format string.

Reply:
Hi, just saw your reply, I've been away.

It's done. I also removed unused KeychainTrustApplication from config.go